
The following have been reloaded with a version change:
  1) hdf5/1.8.18_parallel_studio-2017.1 => hdf5/1.8.18
  2) szip/2.1.1_parallel_studio-2017.1 => szip/2.1.1


The following have been reloaded with a version change:
  1) zlib/1.2.11 => zlib/1.2.11_parallel_studio-2017.1


The following have been reloaded with a version change:
  1) hdf5/1.8.18 => hdf5/1.8.18_parallel_studio-2017.1
  2) szip/2.1.1 => szip/2.1.1_parallel_studio-2017.1
  3) zlib/1.2.11_parallel_studio-2017.1 => zlib/1.2.11


The following have been reloaded with a version change:
  1) hdf5/1.8.18_parallel_studio-2017.1 => hdf5/1.8.18
  2) openmpi/2.0.1_parallel_studio-2017.1 => openmpi/2.1.2_gcc-4.8.5
  3) python/3.7.0 => python/2.7.14_openmpi-2.1.2
  4) szip/2.1.1_parallel_studio-2017.1 => szip/2.1.1

Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
      File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    from asynch_py.asynch_interface_parallel import Assim
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>

    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
: invalid syntax
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
        return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
        run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
        exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
    from asynch_py.asynch_interface_parallel import Assim
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    from asynch_py.asynch_interface_parallel import Assim
  File "Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
        run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
       ^
SyntaxError: invalid syntax
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
           run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
: invalid syntax
       ^
SyntaxError: invalid syntax
^
SyntaxError: invalid syntax
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
        main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
"__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
    "__main__", fname, loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    run_command_line(args)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
    run_path(sys.argv[0], run_name='__main__')
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 252, in run_path
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
        exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 82, in _run_module_code
exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "asynchdist_assim_parallel_simp.py", line 5, in <module>
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line   File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
    from asynch_py.asynch_interface_parallel import Assim
  File "/Users/apensoneault/asynch_python/asynch_py/asynch_interface_parallel.py", line 576
    else:
       ^
SyntaxError: invalid syntax
